subscriptions/admin.py

```diff
@@ -5,16 +5,42 @@
 from subscriptions.conf import SETTINGS
 
 
-class PlanCostInline(admin.TabularInline):
-    """Inline admin class for the PlanCost model."""
-    model = models.PlanCost
+# class PlanCostInline(admin.TabularInline):
```

Comment:
Could we get a note explaining the commented out code?  Any future plans with it?